lib/fetch/index.js

```diff
@@ -189,8 +189,7 @@ async function fetch (input, init = {}) {
       // 4. Abort the fetch() call with p, request, responseObject,
       //    and requestObject’s signal’s abort reason.
       abortFetch(p, request, responseObject, requestObject.signal.reason)
-    },
-    { once: true }
```

Comment:
What about this?

Reply:
the native `events.addAbortListeners` adds it, and the fallback to addEventListener has it as well:
https://github.com/nodejs/undici/blob/479c99d2331baa9eff074c308542d9ee43257ac3/lib/core/util.js#L436
https://github.com/nodejs/node/blob/48345d0f6212d0c21922b5119c43c035013b7c5e/lib/events.js#L1234